src/tooltip/styles.css.ts

```diff
@@ -285,3 +285,75 @@ export const tooltipCSS = recipe<ReactAriaRecipe<TooltipRenderProps>>({
         },
     },
 });
+
+/** -----------------------------------------------------------------------------
+ * Trigger Button
+ * ------------------------------------------------------------------------------- */
+
+export const tooltipFieldButtonCSS = style([
+    css({
+        borderRadius: "sm",
+
+        color: "text_low_contrast",
+        flexShrink: "0",
+        fontStyle: "bodyMd",
+
+        height: "100%",
+        width: "100%",
+    }),
+    withPrefersMotion({
+        transitionDuration: vars.transitionDuration.short,
+        transitionProperty: "background, color",
+        transitionTimingFunction: vars.ease.quart_in_out,
+    }),
+    {
+        selectors: {
+            /**
+             * Ensure spacing between field-buttons within a group.
+             */
+            "&:first-of-type": {
+                marginLeft: vars.spacing.space_1,
+            },
+            "&:last-of-type": {
+                marginRight: vars.spacing.space_1,
+            },
+
+            /**
+             * Whether the button is disabled.
+             */
+            "&[data-disabled]": {
+                opacity: 0.5,
+            },
+            /**
+             * Whether the button is keyboard focused.
+             */
+            "&[data-focus-visible]": {
+                background: vars.color.bg_button_secondary_active,
```

Comment:
I suspect that we may not want a background color here — but let's leave it in for now and assess once deployed